### PR TITLE
0000 Changes for classes and architecture of routes

### PR DESCRIPTION
**Pull request by diana-k22**
Changed the class to classname in the header component. The architecture of routes has been changed.

### CODE REVIEW

File: src/components/MainPage/header/Header.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ const Header = () => {
                     <Link className={s.navLink} to='/'>Как это работает</Link>
                 </nav>
                 <div className={s.headerInput}>
-                    <input class={s.headerSearch} type="text" placeholder='Поиск'/>
```
Comment (Capsonchik) [inline src/components/MainPage/header/Header.jsx:19]: Ля пасиба за фиксы =) С меня печенька =)

Comment (diana-k22) [reply src/components/MainPage/header/Header.jsx:19]: Sps

**Review by Capsonchik — COMMENTED** [review]
Ваще шикака

**Review by Capsonchik — APPROVED** [review]
Ауууфффф

**Review by Cra7yy — APPROVED** [review]
Умничка

**Pull request merged**